src/functions/assignment.js

```diff
@@ -0,0 +1,79 @@
+//@flow
```

Comment:
The name of this file feels a bit misleading.  Could you renamed it to `internal.js`?

Reply:
I think `assignment` refers to `\def` and `\gdef` which is the purpose of this file.  (The idea is that other commands might also generate `internal`.)  I could see calling it `def.js`.  The name `assignment.js` is presumably future-proofing for an eventual `\let` definition? But I'd still lean toward `def.js`.

Reply:
The reason why I suggested internal.js was that it also handles `\global`, but if `\global` can only be used with `\def` then def.js should be fine as well.

Reply:
Changed to `def.js`.